core/cap-0056.md

```diff
@@ -90,7 +90,7 @@ above, except you would match against the `TransactionResult` hash stored in
 TODO: Does this work with InnerTransactionResultPair?
 
 ### XDR Changes
-```
+```diff
```

Comment:
This should actually have an `mddiffcheck.base` specified (look in other CAPs for context). @sisuresh you know how you produced this diff, so maybe you can fix this up?